command/server/listener.go

```diff
@@ -97,6 +97,15 @@ func listenerWrapTLS(
 		}
 		tlsConf.PreferServerCipherSuites = preferServer
 	}
+	if v, ok := config["tls_require_valid_client_cert"]; ok {
```

Comment:
For all the other options, we are using names of the fields in tlsConf as it is with underscores. Can we do the same for this as well?